### PR TITLE
Add WorkflowRun and Workflow to DeploymentEvent

### PR DESCRIPTION
**Pull request by soniachikh**

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -183,7 +185,6 @@ type DeploymentStatusEvent struct {
 	Deployment       *Deployment       `json:"deployment,omitempty"`
 	DeploymentStatus *DeploymentStatus `json:"deployment_status,omitempty"`
 	Repo             *Repository       `json:"repository,omitempty"`
-
```
Comment (gmlewis) [inline github/event_types.go:186]: Let's please leave this blank line as a separator for the next section.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @soniachikh .
Please run `go generate ./...` as described in CONTRIBUTING.md and push the results to this PR.

File: github/event_types.go (hunk 1 above)
Comment (gmlewis) [reply github/event_types.go:205]: See how the blank line moved?  Can you please revert this change?

line-moved-2023-08-15_16-22-43

**Review by gmlewis — CHANGES_REQUESTED** [review]

**Comment by gmlewis** [comment]
@google-ospo-team - here is another PR where the googlebot has failed to comment.  Please address.

**Review by gmlewis — APPROVED** [review]
Thank you, @soniachikh !
LGTM.

Awaiting response from @google-ospo-team before merging.

**Comment by gmlewis** [comment]
Ah!  We now have the cla/google check - @soniachikh - please follow the instructions to sign the Google CLA and then we can merge this PR.  Thank you!

**Comment by google-admin** [comment]
FYI - there is an ongoing issue with GitHub webhooks (https://www.githubstatus.com/) that is likely causing the delays in the CLA check running.

**Comment by stockmaj** [comment]
@gmlewis is there a plan to release soon?  I see it's been a while.

**Comment by gmlewis** [comment]
> @gmlewis is there a plan to release soon? I see it's been a while.

Yes, you are right, and we've had a lot of activity in the repo since the last release.  
It would be nice to get #2865 and #2869 reviewed and merged... and maybe even #2876 .
If you could assist by reviewing the outstanding PRs, that would help tremendously.
Thank you!

**Comment by gmlewis** [comment]
@stockmaj - here is the new release:
https://github.com/google/go-github/releases/tag/v54.0.0

**Comment by stockmaj** [comment]
Thank you.

**Pull request merged**